platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_dependencies.adoc

```diff
@@ -90,29 +90,37 @@ Adopting clear naming conventions for catalog entries ensures that developers ca
 
 The following guidelines help in naming catalog entries effectively:
 
-1. **Use dashes to separate segments**: Prefer dashes (`-`) over underscores (`_`) to separate different parts of the entry name.
+1. **Use dashes to separate segments**: Prefer hyphen/dashes (`-`) over underscores (`_`) to separate different parts of the entry name.
 +
-*Example*: For `org.apache.logging.log4j:log4j-api`, use `log4j-api`
+Example: For `org.apache.logging.log4j:log4j-api`, use `log4j-api`
 
-2. **Derive the first segment from the project group**: Use a unique identifier from the project's group ID as the first segment.
+2. **Strive to keep two segments maximum**: Try to keep it to two segments to avoid excessive nesting.
 +
-*Example*: For `com.fasterxml.jackson.core:jackson-databind`, use `jackson-databind`
+Example: For `io.airlift:aircompressor`, use `airlift-aircompressor`
 
-3. **Omit redundant segments**: If the group and artifact IDs are the same, avoid repeating them.
+3. **Derive the first segment from the project group**: Use a unique identifier from the project's group ID as the first segment. Do not include the top level domain in the segment (`com`, `org`, `net`, `dev`).
 +
-*Example*: For `io.ktor:ktor-client-core`, use `ktor-client-core`, not `ktor-ktor-client-core`
+Example: For `com.fasterxml.jackson.core:jackson-databind`, use `jackson-databind`
```

Comment:
the message above doesn't really explain why we dropped "core", it might be good to expand what we mean by "unique identifier" in a bit more detail.